bin/proactive-server

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env bash
```

Comment:
this pull request should not contain changes in the proactive-server script. Additionally, this script is generated by the build and should not be versioned in the git repository